Doc/reference/datamodel.rst

```diff
@@ -1342,6 +1342,11 @@ Special read-only attributes
    * - .. attribute:: frame.f_locals
      - The dictionary used by the frame to look up
        :ref:`local variables <naming>`
+       (if the frame refers to a function or comprehension,
+       this may return a write-through proxy object)
+
+       .. versionchanged 3.13
```

Comment:
```suggestion
       .. versionchanged:: 3.13
```

Reply:
(currently I think it's just a ReST comment, which is why it's not being rendered at all ;)